src/edu/psu/sweng500/team8/gui/GridPanel.java

```diff
@@ -73,6 +85,23 @@ public void populatePanel(CellGrid grid, GameSession gameSession) {
             }
         }
     }
+    
+    public void remarkGivenCells(CellGrid grid){
```

Comment:
Why is this necessary? It is only called in the Save button handler, but that shouldn't clear the highlights. In fact I commented it out and it didn't seem to make a difference. suggest removing this method